changelogs/update-composer-add-allow-plugins

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: Update
+
+Adding "allow-plugins" property for composer configuration
```

Comment:
The changelog lint failed. Should we have the PR number? `Adding "allow-plugins" property for composer configuration. #8139`

Reply:
Good call @chihsuan ! Done in https://github.com/woocommerce/woocommerce-admin/pull/8139/commits/b5453be2bc875802f04dd281ba0be20e1d8c28eb